src/app/components/map-item-bar/map-item-bar.component.ts

```diff
@@ -78,21 +70,37 @@ export class MapItemBarComponent implements OnInit, OnDestroy {
     return item.FormNames.join(', ');
   }
 
+  getAdditionalAttachmentsCount(count: number): number {
+    return count > 1 ? count - 1 : null;
+  }
+
+  handleMissingImage() {
+    this.firstAttachment = './assets/images/broken-image-w-bg.svg';
+    this.showAttachmentCount = false;
+  }
+
+  private checkIFirstAttachmentUrlIsNotEmpty(url: string): SafeUrl {
```

Comment:
Fortsatt villedende funksjonsnavn. Hva er hensikten med funksjonen?

Reply:
Jeg ser du fjernet funksjonen helt i siste commit. Jeg har ikke noe greie på det, men bør vi kjøre dette på url'en fortsatt?  this.sanitizer.bypassSecurityTrustUrl(url)